### PR TITLE
[WIP] Denial Constraints

### PR DESCRIPTION
**Pull request by clarapueyoballarin**
Added new .dml script for denial constrains calculation
Added its implementation for the builtins

### CODE REVIEW

File: src/main/java/org/apache/sysds/common/Builtins.java [hunk 1]
```diff
@@ -99,6 +99,7 @@
 	COR("cor", true),
 	DBSCAN("dbscan", true),
 	DETECTSCHEMA("detectSchema", false),
+	DENIALCONSTRAINS("denialConstrains", false),
```
Comment (Shafaq-Siddiqi) [inline src/main/java/org/apache/sysds/common/Builtins.java:102]: change the values of the script parameter to true.

**Comment by Shafaq-Siddiqi** [comment]
Hi,
Thank you for your contribution. Please add the tests in our test suite for your built-in. For the DML scripts, we use two-space indentation. If you are still working on the project please mark it as "WIP".

**Review by corepointer — CHANGES_REQUESTED** [review]
* Thanks for the PR, already looks promising. Some code to test it, as @Shafaq-Siddiqi  already said, would be very appreciated :) Have a look at other builtins test functions do it and create a test class like the ones in src/test/java/org/apache/sysds/test/functions/builtin/*
* In addition to the indentation, some variables could have a more descriptive name.
* Please correct constrains to constraints.

**Comment by Shafaq-Siddiqi** [comment]
LGTM,
Thank you @clarapueyoballarin for your contribution. During the merge, I have fixed some minor formatting, warnings, and header of the DML script.

**Pull request closed without merging**